tests/system/Database/Live/ModelTest.php

```diff
@@ -709,17 +728,21 @@ public function testCanCreateAndSaveEntityClasses()
 		$this->assertEquals('Developer', $entity->name);
 		$this->assertEquals('Awesome job, but sometimes makes you bored', $entity->description);
 
+		$time = time();
```

Comment:
Using seconds can cause false negatives when the test execution crosses the threshold from one second to another, which is why we were using just the date here.

Reply:
But job table has a datetime column datatype as integer, so we cannot save the formatted date in it. Hence changed.

Reply:
Then check around the base test classes. I believe @jim-parry create a "close enough" assertion for times like this so tests don't fail randomly because they crossed into a new second during execution.

Reply:
I will just convert this timestamp and database saved timestamp to dates to compare. Will this work?

Reply:
that also works.

Reply:
No problem. Will do it.